easybuild/easyconfigs/c/Code_Saturne/Code_Saturne-5.0.9-intel-2018a.eb

```diff
@@ -0,0 +1,52 @@
+# easyconfig file for Code_Saturne
+# author: Metin Cakircali (Juelich Supercomputing Centre)
+name = 'Code_Saturne'
+version = '5.0.9'
+
+# extra option for the SLURM batch system
+slurm = True
+
+homepage = 'https://www.code-saturne.org'
+description = """Code_Saturne solves the Navier-Stokes equations 
+for 2D, 2D-axisymmetric and 3D flows, steady or unsteady, 
+laminar or turbulent, incompressible or weakly dilatable, 
+isothermal or not, with scalars transport if required.
+
+Code_Saturne %(version)s%(versionsuffix)s is installed in
+$EBROOTCODE_SATURNE
+"""
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+sources = [{
+    'source_urls': ['https://github.com/code-saturne/code_saturne/archive/'],
+    'filename': 'v%(version)s.tar.gz'
+}]
+checksums = ['6166c710a17581e1745af1d0816a38da1dc02000fac885e7c6b9b048ea50444e']
+
+builddependencies = [
+    ('Autotools', '20170619'),
+    ('gettext', '0.19.8.1'),
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.4'),
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('PyQt', '4.12.1', '-Python-%(pyver)s'),
```

Comment:
This version of `PyQt` does not exist in `intel-2018a`. Maybe you missed it in the commit?

Reply:
I guess the issue is that I don't have default EB system to test. Any suggestion for that?

Reply:
use foss?

Reply:
add PyQt intel/2018a?